From 6fe9f93af690363b6e6e21550b3a1dc4793d81ae Mon Sep 17 00:00:00 2001
From: Sanyog Kale <skale@habana.ai>
Date: Sun, 03 Jan 2021 19:09:54 +0200
Subject: [PATCH] [SW-0] goya2_arc: add i2c slave default intr mask macro

Currently, I2C slave interrupt mask macro is added per platform header
file. Since the interrupt mask macro is used in common i2c slave
file, the compilation of ppboot for goya2_arc HL target fails. This
patch adds I2C slave interrupt mask macro for goya2 HL target.

Change-Id: Ib98784d3028f89d4d22cb112565cdd8fac4aad3c
---

diff --git a/goya2_arc/common.h b/goya2_arc/common.h
index 64426a1..2c8cc9f 100644
--- a/goya2_arc/common.h
+++ b/goya2_arc/common.h
@@ -36,6 +36,7 @@
 #define FUSE_WORDS_PER_BANK	0x40
 #define I2C_SLAVE_ADDRESS	0x6A
 #define I2C_SLAVE_ADDRESS_DBG	I2C_SLAVE_ADDRESS
+#define I2C_SLAVE_DEFAULT_INTR_MASK	0x48FF
 
 #define EFUSE_SYSCFG_DISABLE_TWI_IF_MASK        BIT(19)
 #define EFUSE_SYSCFG_FW_VERSION_LOCATION_SPI	BIT(14)
